### DISCUSSION TITLE
Adding the app to F-Droid

### DISCUSSION THREAD

**Discussion by aartoni** [tughi/aggregator-android#48] [Ideas]
**Is your feature request related to a problem? Please describe.**
Currently the app is only available from the Google Play Store, users who are not willing to depend on Google services must therefore install an alternative store (e.g., [Aurora Store](https://auroraoss.com/download/AuroraStore/)) or a replacement for Google's proprietary libraries (e.g., [microG](https://microg.org/)).

**Describe the solution you'd like**
For convenience it would be preferable to be able to download (and update) the application from the [F-Droid](https://f-droid.org/) repositories.

**Describe alternatives you've considered**
Since some developers run into problems trying to add their app, you could also use a [dedicated repository](https://f-droid.org/en/docs/Setup_an_F-Droid_App_Repo/) (with binaries only) instead of the official F-Droid one.

**Comment by ghost** [comment]
Devs seems to be offline & looks like this project needs to be forked...

**Comment by tughi** [comment]
What makes you believe that a fork would save this project?

I haven't received a single PR while actively developing on it.

**Comment by ghost** [comment]
Alas I apologize. So you're active. Good to atleast know that.😅

**Comment by tughi** [comment]
@aartoni, F-Droid sounds interesting. Let's see if I can adapt this repo to be compatible.

**Comment by tughi** [comment]
I submitted a request to add the app to F-Droid: https://gitlab.com/fdroid/rfp/-/issues/2137

**Comment by tughi** [comment]
Aggregator is available on F-Droid: https://f-droid.org/en/packages/com.tughi.aggregator/
